<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDataBaseConnectionWidget</class>
 <widget class="QWidget" name="CDataBaseConnectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Подключение к базе данных</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="0" column="0">
       <widget class="QGroupBox" name="gbxConnectionType">
        <property name="title">
         <string>Расположение базы данных</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QRadioButton" name="rbnDbServer">
           <property name="text">
            <string>На сервере</string>
           </property>
           <property name="icon">
            <iconset resource="connectionwidget_res.qrc">
             <normaloff>:/images/connectionServer.png</normaloff>:/images/connectionServer.png</iconset>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="rbnDbFile">
           <property name="text">
            <string>В файле</string>
           </property>
           <property name="icon">
            <iconset resource="connectionwidget_res.qrc">
             <normaloff>:/images/connectionFile.png</normaloff>:/images/connectionFile.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="chbxDbRemember">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Указанные данные будут сохранены в случае успешного подключения&lt;br/&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;(пароль хранится в зашифрованном виде)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Запомнить</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="1" column="2">
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Пароль</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QLineEdit" name="leDbPassword">
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QGroupBox" name="groupBox_5">
              <property name="title">
               <string>Имя базы данных</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="0">
                <widget class="QLineEdit" name="leDbName"/>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Логин</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <widget class="QLineEdit" name="leDbLogin"/>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Хост или IP адрес</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QComboBox" name="cbxDbHost">
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Файл базы данных</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <widget class="QComboBox" name="cbxDbFileName"/>
             </item>
             <item row="0" column="1">
              <widget class="QToolButton" name="tbnDbFile">
               <property name="toolTip">
                <string>Выбрать файл базы данных</string>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>101</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbnDbExit">
            <property name="text">
             <string>Выход</string>
            </property>
            <property name="icon">
             <iconset resource="connectionwidget_res.qrc">
              <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="lDbConnectionStatus">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbnDbConnect">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>Подключиться</string>
            </property>
            <property name="icon">
             <iconset resource="connectionwidget_res.qrc">
              <normaloff>:/images/arrow_right.png</normaloff>:/images/arrow_right.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pbnDbConnect</tabstop>
  <tabstop>cbxDbHost</tabstop>
  <tabstop>leDbName</tabstop>
  <tabstop>leDbLogin</tabstop>
  <tabstop>leDbPassword</tabstop>
  <tabstop>chbxDbRemember</tabstop>
  <tabstop>rbnDbServer</tabstop>
  <tabstop>rbnDbFile</tabstop>
  <tabstop>pbnDbExit</tabstop>
  <tabstop>tbnDbFile</tabstop>
  <tabstop>cbxDbFileName</tabstop>
 </tabstops>
 <resources>
  <include location="connectionwidget_res.qrc"/>
  <include location="connectionwidget_res.qrc"/>
  <include location="connectionwidget_res.qrc"/>
  <include location="connectionwidget_res.qrc"/>
 </resources>
 <connections/>
</ui>
